docs/architecture/decisions/0007-python-buildpack.md

```diff
@@ -0,0 +1,28 @@
+# 7. Python Buildpack
+
+Date: 2022-08-12
+
+## Status
+
+Accepted
+
+## Context
+
+We had previously drafted ADRs to use Docker to build images for containerized deployment. The motivation was to reduce dependency on Cloud.gov. These ADRs were rejected, mainly owing to the necessity of having an image repository to hold the built images. The team had concerns about an overreliance on GitHub with their GitHub Packages service and about the overall maintenance burden of any image repository.
+
+Cloud.gov uses Cloud Foundry which provides several “buildpacks”. These are automated environments which will take a code repository of a certain language and do the usual setup steps to prepare a deployment of that code. In the case of Python, this means automated detection of Pipfile and installation of packages.
+
```

Comment:
Want to add that we didn't think we'd need a "custom" buildpack since the current use case is so simple, as a reason too?